oeps/best-practices/oep-0051-bp-conventional-commits.rst

```diff
@@ -221,6 +230,8 @@ Some commit guidelines are prescriptive about what grammar to use in commit subj
 Change History
 ==============
 
+2021-11-01: Not exceptions like ``squash! `` and ``fixup! ``.
```

Comment:
```suggestion
2021-11-08: Note exceptions like ``squash! `` and ``fixup! ``.
```

Reply:
The space at the end of the tick-tick is what made the build fail... weird.

Reply:
Thanks. Updated.